Jenkinsfile

```diff
@@ -4,6 +4,8 @@
 
 // mxnet libraries
 mx_lib = 'lib/libmxnet.so, lib/libmxnet.a, dmlc-core/libdmlc.a, nnvm/lib/libnnvm.a'
+// mxnet cmake libraries
```

Comment:
Specify why we're removing nvmm. This could confuse people

Reply:
Good point.  Will update.

Reply:
Updated.